staging/src/k8s.io/apiserver/pkg/apis/config/v1/defaults.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	defaultTimeout         = &metav1.Duration{Duration: 3 * time.Second}
+	defaultCacheSize int32 = 1000
+)
+
+func addDefaultingFuncs(scheme *runtime.Scheme) error {
+	return RegisterDefaults(scheme)
+}
+
+// SetDefaults_EncryptionConfiguration applies defaults to EncryptionConfiguration.
+func SetDefaults_EncryptionConfiguration(obj *EncryptionConfiguration) {
+	for _, r := range obj.Resources {
+		for _, p := range r.Providers {
+			if p.KMS != nil {
+				setKMSProviderDefaults(p.KMS)
+			}
+		}
+	}
+}
+
+func setKMSProviderDefaults(obj *KMSConfiguration) {
```

Comment:
name this `SetDefaults_KMSConfiguration`, and defaulting generation will handle calling it, so you don't need to define `SetDefaults_EncryptionConfiguration`

Reply:
Done.